mln/admin/__init__.py

```diff
@@ -100,9 +106,9 @@ def get_settings_inlines(obj):
 friend_cond_admin.search_fields = "networker", "condition", "success_body__subject", "success_body__text", "failure_body__subject", "failure_body__text", "source__source"
 
 trigger_admin = make_inline(NetworkerMessageTrigger, NetworkerMessageAttachment)
-trigger_admin.list_display = "networker", "body", "trigger", "source", "notes"
-trigger_admin.list_display_links = "body",
-trigger_admin.search_fields = "networker", "body__subject", "body__text", "trigger", "source", "notes"
+trigger_admin.list_display = "networker", "response"
+trigger_admin.list_display_links = "response",
+trigger_admin.search_fields = "networker", "response__subject", "response__text"
 

```

Comment:
If you're keeping TriggerDev around you should also keep this around, and just add another for actual triggers.